bblfsh/compat.py

```diff
@@ -0,0 +1,228 @@
+import os
+import sys
+from typing import Union, List, Any, Optional
+
+import grpc
+
+import bblfsh.client as bcli
+from bblfsh import role_id, role_name
+from bblfsh.node import Node
+from bblfsh.node_iterator import NodeIterator
+from bblfsh.result_context import ResultContext
+from bblfsh.aliases import (
+    ParseRequest, ParseResponse, DriverStub, ProtocolServiceStub,
+    VersionRequest, SupportedLanguagesRequest, ModeType,
+    Mode, VersionResponse, DESCRIPTOR
+)
+from bblfsh.pyuast import uast, iterator as native_iterator
+from bblfsh.tree_order import TreeOrder
+
+if "BBLFSH_COMPAT_SHUTUP" not in os.environ:
```

Comment:
Haha, this name. 😁

Unless someone requested this, I'd be inclined to post the warning unconditionally. Otherwise this will get wired into somebody's setup script somewhere and never touched again. If someone comes requesting a silence for it, we can use that as an opportunity to work with them to get them migrated.

Reply:
Someone will probably request it... but let's remove it until then.